mne/viz/epochs.py

```diff
@@ -750,6 +749,12 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20, n_channels=20,
         at least a 120 Hz displayed sample rate).
 
         .. versionadded:: 0.15
+    noise_cov : instance of Covariance | str | None
+        Noise covariance used to whiten the data while plotting.
+        Whitened data channels are scaled by ``scalings['whitened']``,
+        and their channel names are shown in italic.
```

Comment:
why the str option?

Reply:
now I see that it is to be able to read from file from disk.
Please document it.